examples/seismic/tutorials/04_dask.ipynb

```diff
@@ -324,11 +342,17 @@
     "\n",
     "    geometry, rec = load_shot_data(param['shot_id'], dt)\n",
     "    geometry.model = model0\n",
-    "    # Set up solver.\n",
-    "    solver = AcousticWaveSolver(model0, geometry, space_order=4)\n",
+    "    # Set up solver -- load the solver used above in the genration of the syntethic data.    \n",
+    "    with open(\"arguments.pkl\", \"rb\") as cp_file:\n",
+    "        cp = pickle.load(cp_file)\n",
+    "        \n",
+    "    op = cp['operator']\n",
+    "    op.model=model0\n",
+    "    op.geometry=geometry\n",
+    "    op.dt=dt\n",
```

Comment:
I would like it better if the part about synthetic data was inside a self-contained `load_shot(i)` function/method that  can do synthetic data in the example, and can be replaced with actual loading from a file for a more realistic setup. What do other people think?

Reusing the solver that was used to generate the synthetic data, for the actual inversion, seems a bit backwards in that sense.

Reply:
The notebook would be much more organized I think, but as @ggorman ever said, the notebooks are given for illustrative purposes only. Someone could adapt it for your own use case and can do so better than us.